src/Transition/VCompareGlobalRoutines.f90

```diff
@@ -799,31 +808,32 @@ SUBROUTINE CheckSpecialObjects(DifUnit,ObjectName,CurArgs,OutArgs,FieldNames,Fie
           CALL WriteOutIDFLinesAsSingleLine(DifUnit,ObjectName,CurArgs,OutArgs,FieldNames,FieldUnits)
 
         CASE('BUILDINGSURFACE:DETAILED')
+          NVertFieldNum = 10 + SurfaceSpaceFieldShift
           IF (MakeUPPERCase(OutArgs(11)) == 'AUTOCALCULATE') THEN
```

Comment:
What about this line? "11" has been replaced with NVertFieldNum and line 812 has OutArgs(11). For input files > V9.2 and < V9.6, this would be OutArgs(10), right? So OutArgs(NVertFieldNum) here as well.

Reply:
Well, this does convert a V9.4 file to V9.5 so maybe this is OK.

![image](https://user-images.githubusercontent.com/5778156/131851817-0f80ec9d-3e3f-45c3-8a87-8fbdd3d37a8f.png)

Reply:
@rraustad Good catch. I can fix shortly, or feel free to push a fix yourself.

Reply:
Oh, it's when the number of vertices field is AutoCalculate when there is a problem. So this does need to change.

![image](https://user-images.githubusercontent.com/5778156/131852574-3b118825-32f3-41f0-ae6a-6fd3638167cc.png)

Reply:
Try that same file with Number of Vertices = autocalculate.

Reply:
I'll build and test this again before I commit.

Reply:
I'm not sure, I was just updating an UnmetHours file and saw that. I actually think it was an ill formed input file. I guess I should also open the transitioned file in IDFEditor for this test as well. Can't hurt to check everything we can think of.

Reply:
Transition worked but opening transitioned file in IDF Editor shows an error. 

![image](https://user-images.githubusercontent.com/5778156/131857176-bb4e6739-a779-429c-beef-3b891a98ea73.png)

Here is the transitioned OA Sys object. This is V9.4 -> V9.5.

  AirLoopHVAC:OutdoorAirSystem,
    Outside Air System,      !- Name
    Outside Air System Controllers,  !- Controller List Name
    Outside Air System Equipment,  !- Outdoor Air Equipment List Name
    Evap Cooler Availability List;  !- Availability Manager List Name

Reply:
Sorry, was using V9.6 IDF Editor.

Reply:
OK, the above issue is resolved using V9.5 IDF Editor and the transition above for V9.5 AirLoopHVAC:OutdoorAirSystem is correct.